<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrearPartida</class>
 <widget class="QMainWindow" name="CrearPartida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>8</y>
       <width>425</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../client/vista/data/background1.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>641</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="mapaComboBox">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>140</y>
       <width>65</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	background-color: white;
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	font: bold 14px;
}

QComboBox QAbstractItemView {
	background: white;
	background-color: darkgray;
	border-style: outset;
	border-width: 2px;
	border-color: grey;
	font: bold 14px;
	color: black;
	border: 2px solid darkgray;
}

QComboBox::drop-down {
	border-left-width: 1px;
	border-left-color: darkgray;
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}

QComboBox::item:hover {
	color: black;
	background-color: gray;
}</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Mapa</string>
     </property>
     <item>
      <property name="text">
       <string>Mapa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="nuevaPartidaButton">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>40</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: white;
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	font: bold 14px;
}

QPushButton:hover {
	background: grey;
}

QPushButton:pressed {
	background-color: grey;
	border-style: inset;
}</string>
     </property>
     <property name="text">
      <string>Nueva partida</string>
     </property>
    </widget>
    <widget class="QComboBox" name="jugadoresComboBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	background-color: white;
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	font: bold 14px;
}

QComboBox QAbstractItemView {
	background: white;
	background-color: darkgray;
	border-style: outset;
	border-width: 2px;
	border-color: grey;
	font: bold 14px;
	color: black;
	border: 2px solid darkgray;
}

QComboBox::drop-down {
	border-left-width: 1px;
	border-left-color: darkgray;
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}

QComboBox::item:hover {
	color: black;
	background-color: gray;
}</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Cantidad de jugadores</string>
     </property>
     <item>
      <property name="text">
       <string>Cantidad de jugadores</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
